cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,57 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/alerts')
   });
 
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (str) => {
+    expect(str).to.equal(`You clicked a button`)
+    })
   });
 
   it('should have the ability to assert scheduled allert', () => {
-
+    cy.get('#timerAlertButton')
+      .click();
+    cy.wait(5000);
+    cy.on('window:alert', (str) => {
+    expect(str).to.equal(`This alert appeared after 5 seconds`)
+    })
   });
 
-  it('should autimatically resolve alerts', () => {
-
-  });
+  it('should automatically resolve alerts', () => {
+    cy.get('#confirmButton')
+      .click();
+    cy.on('window:confirm', (str) => {
```

Comment:
The test lacks an assertion for the scenario when the confirm doesn't appear. Add an assertion to confirm that the confirm appears before proceeding with the rest of the test.